contracts/ColonyTask.sol

```diff
@@ -89,13 +89,20 @@ contract ColonyTask is ColonyStorage {
     _;
   }
 
-  function makeTask(bytes32 _specificationHash, uint256 _domainId, uint256 _skillId, uint256 _dueDate) public
+  function makeTask(
+    uint256 _parentDomainId,
+    uint256 _domainProofIndex,
+    bytes32 _specificationHash,
+    uint256 _domainId,
+    uint256 _skillId,
+    uint256 _dueDate
+  ) public
   stoppable
-  auth
+  auth2(_parentDomainId, _domainId, _domainProofIndex)
```

Comment:
Not necessarily, since the permission might be held in a parent domain. If I have Administration permission in domain A, and then I try to create a task in child domain B, we would need to provide the proof that B is a child domain of A.